epochtalk/epochtalk
# Pm reports

Test Procedure:

1) Report a pm

2) Log in as admin and go to the Moderation -> Messages tab

3) Ensure that all actions work on this page. Should work identical to other moderation views.

4) Paging was reverted to use ng-href. Ensure that when paging with a selected report in the Users moderation tab. The user stays selected.

5) Test that the recompile sass script works
